libbeat/publisher/pipeline/ttl_batch_test.go

```diff
@@ -127,6 +133,10 @@ func (b *mockQueueBatch) Entry(i int) queue.Entry {
 	return fmt.Sprintf("event %v", i)
 }
 
+func (b *mockQueueBatch) FreeEntries() {
+	b.freeEntriesCalled++
```

Comment:
This can cause problems if the test code runs in multiple goroutines somehow, might not be a problem rn.